components/topbar/user/src/index.scss

```diff
@@ -164,27 +165,6 @@ $bgc-topbar-user-cta: $c-accent !default;
   }
 
   &-cta {
-    &Button {
-      padding: $p-v 0;
-
-      &Link {
-        @include reset-link;
-        @include sui-button(
-          'custom',
-          'small',
-          $background-color: $bgc-topbar-user-cta,
```

Comment:
This var can be removed: `$bgc-topbar-user-cta`

Reply:
well seen dude!
deleted.